android/src/main/java/ie/gov/tracing/nearby/ExposureNotificationClientWrapper.java

```diff
@@ -98,6 +98,10 @@ public boolean deviceSupportsLocationlessScanning() {
     return exposureNotificationClient.deviceSupportsLocationlessScanning();
   }
   
+  public Task<Long> deviceSupportsENS() {
```

Comment:
Should this be named getDeviceENSVersion

Reply:
will rename